src/eval.jl

```diff
@@ -99,3 +99,21 @@ handle("eval-repl") do data
     end
   end
 end
+
+handle("docs") do data
+  @dynamic let Media.input = Editor()
+    result = withpath(getpath(data)) do
```

Comment:
No, I don't think it should be necessary in this case, nor the Media.input binding. Also, on Julia 0.4 you can switch `@d` for just `d`.